src/main.js

```diff
@@ -132,6 +133,26 @@ module.exports = {
       ignoredRulesWhenFixing = idsToIgnoredRules(ids)
     }))
 
+    this.subscriptions.add(atom.contextMenu.add({
+      'atom-text-editor:not(.mini), .overlayer': [{
+        label: 'ESLint Fix',
+        command: 'linter-eslint:fix-file',
+        shouldDisplay: (evt) => {
+          const activeEditor = atom.workspace.getActiveTextEditor()
+          if (!activeEditor) {
+            return false
+          }
+          // Black magic!
+          // Compares the private component property of the active TextEditor
+          //   against the components of the elements
+          const evtIsActiveEditor = evt.path.some(elem =>
+            elem.component && elem.component === activeEditor.component)
```

Comment:
Note: it seems that `activeEditor` does not have a `component` prop in atom `1.18.0`, so this only works in `1.19.0` and up.  I think that's fine, personally.